app/models/chat_channel.rb

```diff
@@ -166,7 +166,11 @@ def pending_users_select_fields
   end
 
   def self.pm_notifications_channel(user_id)
-    "private-message-notifications--#{ApplicationConfig['APP_DOMAIN']}-#{user_id}"
+    "private-message-notifications--#{urlsafe_encoded_app_domain}-#{user_id}"
+  end
+
+  def self.urlsafe_encoded_app_domain
+    Base64.urlsafe_encode64(ApplicationConfig["APP_DOMAIN"])
```

Comment:
`urlsafe_encode64` is the best option because it uses ‘-’ instead of ‘+’ and ‘_’ instead of ‘/’. Pusher doesn't allow `+`, which would've been included if we use the simple `encode64.

ref https://apidock.com/ruby/Base64/urlsafe_encode64

Reply:
<3